distro/pom.xml

```diff
@@ -327,6 +327,13 @@
                     export BOLD=`tput bold`&#10;export PROJECT_NAME="${project.artifactId}"
                   </replacevalue>
                 </replace>
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/utils.sh"
+                  token='export O3_HOSTNAME="localhost"'>
```

Comment:
Should we also have `:4001` port mentioned here as well?

Reply:
Yes @vasharma05 
This will help us in displaying that table showing `HIS Component`, `URL`, `Username` and `Password`